packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -300,9 +307,25 @@ export default {
       this.validationService.checkEmailInput("email");
       this.validationService.checkFormInput("state");
       this.validationService.checkFormCheckboxInput("consentAgree");
+      this.validationService.checkFormInput("captcha");
 
       if (this.validationService.isValidForm()) {
         // make api call
+        const { baseUri, registrationPolicyId} = this.$site.themeConfig.uris;
+        const registrationPath = `/api/v1/registration/${registrationPolicyId}/register`;
+        const body = {
+          userProfile: {
+            email: this.form.email.value,
+            firstName: this.form.firstName.value,
+            lastName: this.form.lastName.value,
+            country: this.form.country.value,
+            state: this.form.state.value,
+            emailOptInC: this.form.consentAgree.value,
+            captchaResponse: this.form.captcha.value,
+          },
+        };
```

Comment:
just a minor note, in the next commits, probably will nice to wrap this part of code in some specific function and not to store it here